server/routes/user.js

```diff
@@ -152,7 +152,7 @@ router.get("/currentUser", authenticate, (req, res) => {
 });
 
 //Get user by id
-router.get("/:id", async (req, res) => {
+router.get("/getUser/:id", async (req, res) => {
```

Comment:
This router is on the `/user` path, so I think it was fine as is. My `dbCamelCase` branch already reverts this since it's updating route names as well. I can change it back to `/get-user/:id` if you still think that would be better though.

Reply:
I changed it back to just /:id. I initially added the /get-user to be much more specific and I had a case where if I had a route after the /:id route that is also the same route but different parameter name, it will use the /:id. If I'm not wrong, I dont think we wont be adding more routes in the user that just uses a parameter.